js-scripts/getSqrtRatioAtTick.ts

```diff
@@ -0,0 +1,8 @@
+// TODO: once hardhat is gone, js-scripts can go inside test folder with its own package.json
```

Comment:
actually we may want to keep core as an npm package, but without hardhat test, we can put these sscripts in test, otherwise hardhat tries to execute them, couldn't find a way to exclude files...